pages/chaoss/chaoss.py

```diff
@@ -23,14 +23,14 @@
                 dbc.Col(gc_first_time_contributions, width=6),
             ],
             align="center",
-            style={"margin-bottom": ".5%"},
+            style={"marginBottom": ".5%"},
```

Comment:
i think this might be a mistake change but not sure

Reply:
Looking at all below seeing the same thing, do you know what this is?

Reply:
yeah the change was made because the browser was suggesting the name change from the hyphen-separated list to the camel case, and now it's not erroring and is correctly formatted.